execution/gethexec/node.go

```diff
@@ -26,6 +26,36 @@ import (
 	flag "github.com/spf13/pflag"
 )
 
+type DangerousConfig struct {
+	ReorgToBlock int64 `koanf:"reorg-to-block"`
+}
+
+var DefaultDangerousConfig = DangerousConfig{
+	ReorgToBlock: -1,
+}
+
+func DangerousConfigAddOptions(prefix string, f *flag.FlagSet) {
+	f.Int64(prefix+".reorg-to-block", DefaultDangerousConfig.ReorgToBlock, "DANGEROUS! forces a reorg to an old block height. To be used for testing only. -1 to disable")
+}
+
+type StylusTargetConfig struct {
+	Arm  string `koanf:"arm"`
+	X86  string `koanf:"x86"`
+	Host string `koanf:"host"`
+}
+
+var DefaultStylusTargetConfig = StylusTargetConfig{
+	Arm:  "arm64-linux-unknown+neon",
+	X86:  "x86_64-linux-unknown+sse4.2",
+	Host: "aarch64-apple-darwin",
```

Comment:
default string for Host should be empty, and in target_from_string, if string is empty we should return Target::default() (which is local machine).